app/src/main/java/com/example/bigowlapp/activity/SendSmsInvitationActivity.java

```diff
@@ -18,18 +19,38 @@
 public class SendSmsInvitationActivity extends AppCompatActivity {
 
     private EditText number, message;
-    private Button send, testSendRequest;
-    private String number2, message2;
+    private Button send;
+    private String number2, message2, noteText;
+    private TextView noteTv;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_send_sms_invitation);
 
-        number = findViewById(R.id.number);
+        noteTv = (TextView) findViewById(R.id.note);
+
+        String contactDetails = getIntent().getStringExtra("number1");
+        String trimContactNumber = "000";
+        if (contactDetails != null) {
+            trimContactNumber = contactDetails.replaceAll("\\D+", "");
+        }
+        String contactNumber;
+        if(trimContactNumber.length()>=10){
+            contactNumber = trimContactNumber.substring(trimContactNumber.length() - 10);
+        }
+        else contactNumber = trimContactNumber;
```

Comment:
Could you explain what's going on here. Just curious.

Also, please fix style in this file. can just use `CTRL+ALT+SHIFT+L`

Reply:
ok. I can fix style. This is to take the the last 10 digits of the input of the dialing  Edit text and use it as the text sms sending phone number